app/src/main/java/com/example/tasky/agenda/domain/model/Attendee.kt

```diff
@@ -6,5 +6,7 @@ data class Attendee(
     val userId: String = "",
     val eventId: String? = null,
     val isGoing: Boolean? = null,
-    val remindAt: Long? = null
+    val remindAt: Long? = null,
+    val isUserEventCreator: Boolean = false,
+    val initials: String = ""
```

Comment:
The initials are always dependent on the full name, so it doesn't need to be a separate property since it can always be computed with that. What you could do though is to make the initials a member variable of the class with a getter:
```
data class Attendee(...) {
    val initials: String get() = fullNameToInitials(fullName)
}
```
Otherwise, you might run into bugs when updating the full name property which doesn't automatically update the initials property (not going to be an issue in this case, but there are comparable scenarios)